package.json

```diff
@@ -36,14 +36,19 @@
   "homepage": "https://github.com/aws-amplify/maplibre-gl-js-amplify#readme",
   "devDependencies": {
     "@aws-amplify/core": "^4.3.1",
-    "@aws-amplify/geo": "^1.1.1",
+    "@aws-amplify/geo": "geo",
```

Comment:
Should we pin a version that we know works with this release? There could be some breaking changes before GA in amplify-js library

Reply:
I'll update to whatever the current working version is